### ISSUE TITLE
Tests in error: JpegImageTest.setUp:36 » NoClassDefFound Could not initialize class org.libjpe...

### ISSUE THREAD

**Issue by VonUniGE** [dbmdz/iiif-server-hymir#4]
### Observed behavior:
Running `mvn clean install` produces:
```
Tests in error:
  JpegImageTest.setUp:36 » NoClassDefFound Could not initialize class org.libjpe...
```
### Desired behavior
No errors in test

### Steps to reproduce
On Debian 8 (Jessie) or on macOS, **without libturbojpeg installed**: 
Run:
```
mvn clean install
```

**Comment by VonUniGE** [comment]
The problem comes from LibraryLoader.java:
``` String filename = System.mapLibraryName(name);
      InputStream in = LibraryLoader.class.getClassLoader().getResourceAsStream(filename);
      int pos = filename.lastIndexOf('.');
      File file = File.createTempFile(filename.substring(0, pos), filename.substring(pos));
      file.deleteOnExit();
      try {
        byte[] buf = new byte[4096];
        OutputStream out = new FileOutputStream(file);
        try {
          while (in.available() > 0) {
            int len = in.read(buf);
            if (len >= 0) {
              out.write(buf, 0, len);
            }
          }
        } finally {
          out.close();
        }
```
If the library is not installed in the system, the variable `in` is null and a NullPointerException is thrown on its first use (`in.available()`).

**Comment by jbaiter** [comment]
Sorry about that, we changed some namespaces and forgot to upload the JNI library with the matching functions. Should be up later today!

**Comment by VonUniGE** [comment]
I suppose that all tests marked with `@Category(JniTest.class)` should not be executed when the library libjpegturbo is not installed?

**Comment by morpheus-87** [comment]
Fixed with a2bb6d9.
